### ISSUE TITLE
is SwiftyEdit free for making company website for commercial use

### ISSUE THREAD

**Issue by bluesky0341** [SwiftyEdit/SwiftyEdit#77]
is SwiftyEdit free for making company website for commercial use

**Comment by patkon** [comment]
Yes. SwiftyEdit is licensed under the GNU General Public License and can therefore be used free of charge for all projects. Of course also for commercial ones.
